packages/tc-ui/src/primitives/multiple-choice/server.jsx

```diff
@@ -87,15 +87,15 @@ const Checkbox = ({
 const EditMultipleChoice = props => {
 	const {
 		propertyName,
-		value,
-		options,
 		disabled,
 		isNested,
 		parentCode,
 		onChange,
 		type,
 	} = props;
 	const name = !isNested ? propertyName : '';
+	const options = Object.keys(getEnums()[type]);
+	const value = props.value ? props.value.sort() : [];
```

Comment:
Should the array be cloned before sorting as this will sort in place rather than create a new array?

Reply:
Yes... but I don't actually know why that is even sorted (again, a port from the old get-value file)

Reply:
I _think_ it was probably copied from the relationship component (where order matters as we want to display them alphabetically ,whereas here they are displayed in the order the enum defines)